DEBUG 2022-10-01 14:33:56.297 UTC: 	 CRAB Client version: v3.220823
DEBUG 2022-10-01 14:33:56.297 UTC: 	 Running on: Linux lxplus792.cern.ch 3.10.0-1160.71.1.el7.x86_64 #1 SMP Tue Jun 28 15:37:28 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-10-01 14:33:56.298 UTC: 	 Executing command: 'submit'
DEBUG 2022-10-01 14:33:56.298 UTC: 	 Found CRAB cache file /afs/cern.ch/user/c/chenyi/.crab3
INFO 2022-10-01 14:33:56.299 UTC: 	 Will use CRAB configuration file Submit.py
DEBUG 2022-10-01 14:33:56.299 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-10-01 14:33:56.304 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-10-01 14:33:56.304 UTC: 	 Working area '/afs/cern.ch/work/c/chenyi/PhysicsWorkspace/HI2022HLT/JetEnergyCorrection/25126_HLTJECGenJets/crab_20221001_GenParticlesRerun' already exists 
Please change the requestName in the config file
ERROR 2022-10-01 14:33:56.304 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220823.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220823.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220823.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220823.00/lib/CRABClient/Commands/SubCommand.py", line 318, in __init__
    self.requestarea, self.requestname, self.logfile = createWorkArea(
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220823.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
CRABClient.ClientExceptions.ConfigException: Working area '/afs/cern.ch/work/c/chenyi/PhysicsWorkspace/HI2022HLT/JetEnergyCorrection/25126_HLTJECGenJets/crab_20221001_GenParticlesRerun' already exists 
Please change the requestName in the config file
INFO 2022-10-01 14:33:56.311 UTC: 	 Log file is /afs/cern.ch/work/c/chenyi/PhysicsWorkspace/HI2022HLT/JetEnergyCorrection/25126_HLTJECGenJets/crab.log
